### PR TITLE
keymap for squiggle goldberg.

### PR DESCRIPTION
**Pull request by ibnuda**
Added a keymap example for squiggle, goldberg version.
## Description

currently, there is no example for `LAYOUT_complicated` for squiggle keyboard. this PR fixes that.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/squiggle/keymaps/rick-complicated/keymap.c [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+void matrix_scan_user() {
+  LEADER_DICTIONARY() {
+    leading = false;
+    leader_end();
+
+    SEQ_ONE_KEY(KC_U) {
+      SEND_STRING(":luvu:");
+      register_code(KC_ENT);
+      unregister_code(KC_ENT);
```
Comment (fauxpark) [inline keyboards/squiggle/keymaps/rick-complicated/keymap.c:194]:
```suggestion
      SEND_STRING(":luvu:\n");
```

Comment (ibnuda) [reply keyboards/squiggle/keymaps/rick-complicated/keymap.c:194]: the intention of that string is sending a single emoji in discord without adding a newline.

Comment (fauxpark) [reply keyboards/squiggle/keymaps/rick-complicated/keymap.c:194]: Yes, and this does the same thing - `\n` is mapped to Enter, it won't make a new line in the text box.

Comment (ibnuda) [reply keyboards/squiggle/keymaps/rick-complicated/keymap.c:194]: Oh, I didn't know that. thanks. please wait 10 minutes.

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**